lib/node_modules/@stdlib/array/base/unary3d-by/test/test.js

```diff
@@ -0,0 +1,286 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var zeros3d = require( '@stdlib/array/base/zeros3d' );
+var numel = require( '@stdlib/ndarray/base/numel' );
+var unary3dBy = require( './../lib' );
+
+
+// FUNCTIONS //
+
+function accessor( v ) {
+	if ( v === void 0 ) {
+		return;
+	}
+	return v * 2.0;
+}
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof unary3dBy, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function applies a function to each nested input array element according to a callback function and assigns results to a nested output array', function test( t ) {
+	var expected;
+	var shape;
+	var x;
+	var y;
+
+	shape = [ 2, 2, 2 ];
+	x = [
+		    [
```

Comment:
JavaScript files should always use TAB indentation. Please ensure you have properly configured EditorConfig, per the development guide.

Reply:
Yea, I'll look into it!